src/test/java/dk/xam/StarfishTest.java

```diff
@@ -0,0 +1,72 @@
+package dk.xam;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+
+import static dk.xam.starfish.Starfish.validate_url;
+import static dk.xam.starfish.Starfish.launch_editor;
+import static dk.xam.starfish.Starfish.isWindows;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+
+@QuarkusTest
+public class StarfishTest {
+
+    @Test
+    public void testURLValidator() {
+        
+        assertEquals(true,validate_url("https://github.com/user-name/repo-name.git"),"URL Validation test failes");
```

Comment:
"failes" is just "fails" - but normally you either leave the description empty or describe what was validated. i.e. `assertEquals(true, validate_url("https://github.com/user-name/repo-name.git"), "Plain github url"))`

Reply:
Ugh a typo 
"Plain github url" : yup!! I'll put in that